### ISSUE TITLE
[Bug] Launch button on app card does not have a hover state 

### ISSUE THREAD

**Issue by ramsaptami** [appsmithorg/appsmith#7621]
## Description
Launch button on the app card on home page does not have a hover state as is the case with the Edit button

### Steps to reproduce the behaviour:
1. Hover over Launch and Edit button on the app card and observe hover state of both

### Important Details
- Version: Cloud
- OS: Win10
- Browser: Chrome
- Environment: Deploy preview (7304)

https://www.loom.com/share/e2879af839244aaea44d906dcf4e2c59

**Comment by VISHAL-143x** [comment]
Can i work on this issue I am a nubie.

**Comment by ramsaptami** [comment]
@VISHAL-143x It's all your's. 

Please read the [Code Contribution Guidelines](https://github.com/appsmithorg/appsmith/blob/master/contributions/CodeContributionsGuidelines.md) to understand the process to get started with solving a Github issue.

**Comment by albinAppsmith** [comment]
Hi @VISHAL-143x, This issue is in `epic/appsmith-design-system-v2` branch. So checkout a new branch from this branch and go ahead.

**Comment by ramsaptami** [comment]
To chime in with some extra info, we currently don't have a design for the hover state of this button but we'd love to see you come up with suitable solution :)
If you feel you'd like something different instead, #7664 is up for grabs if you'd like.

**Comment by JubayerJoy** [comment]
Hi @ramsaptami, I would love to work on this. Let me put in a PR and adjust from there, how about that?
Cheers. 🎈

**Comment by aromalsanthosh** [comment]
Hi @ramsaptami , I can solve this can u assign it to me ?

**Comment by kyteinsky** [comment]
I think the issue has already been solved. I saw the website and the hover property is already present in the launch button.

**Comment by ramsaptami** [comment]
Hey, we've had a design revamp on our product and you'll likely see the described scenario on the new home page. You can give it a go if you're still up for it and I'll assign the issue to whoever's interested.

**Comment by kyteinsky** [comment]
Yeah, I see it now. You can assign it to me.
Just to confirm, the website follows the release branch, right?

**Comment by ramsaptami** [comment]
Yep, that's right.

Don't forget to read the [contributor guidelines](https://github.com/appsmithorg/appsmith/blob/769ee31f87cfe7866e5cd87a2c2df9da48f778cc/CONTRIBUTING.md) before working on this issue. Good luck!

**Comment by kyteinsky** [comment]
> Yep, that's right.
> 
> Don't forget to read the [contributor guidelines](https://github.com/appsmithorg/appsmith/blob/769ee31f87cfe7866e5cd87a2c2df9da48f778cc/CONTRIBUTING.md) before working on this issue. Good luck!

Sure will. Thanks.

**Comment by albinAppsmith** [comment]
@momcilo-appsmith @Debsourabh Please suggest a hover state for the black button.

**Comment by kyteinsky** [comment]
@albinAppsmith Thanks for the feedback. I already saw that file, and it doesn't seem possible that this could be done from there because the edit button and the launch buttons both use the same Button component. Therefore if we make it go dark on hover (the orange Edit button goes dark), it doesn't work with black. Either we need to change the launch button's background color, or we need CSS outside of the button component.

I have attached the change that made the black hover state possible in the picture in the previous comment.
![IMG_20211007_114954_946.png](https://user-images.githubusercontent.com/20724224/136330172-8c5aea22-5764-4541-87b5-7d5f80ca7a69.png)

Would you please suggest what to do?

**Comment by albinAppsmith** [comment]
@kyteinsky  Yes, you can proceed like this since the black button is not a global button or button state we don need to put this in `ads/Button.tsx`, you can extend the style like this and proceed.

**Comment by kyteinsky** [comment]
@albinAppsmith Thanks!

**Comment by kyteinsky** [comment]
@albinAppsmith It would be great if you could spare a minute to review the PR?

**Comment by albinAppsmith** [comment]
@kyteinsky sorry if I missed to notify you, can you try the suggestion which @momcilo-appsmith  gave?

**Comment by kyteinsky** [comment]
Yeah sure, should I open another PR for that or commit to the existing one itself?

**Comment by momcilo-appsmith** [comment]
@kyteinsky This looks good.

**Comment by albinAppsmith** [comment]
@kyteinsky  Sure, Our team will be reviewing the changes you have made and the same will be approved after validation is done.
